docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/sequence/ScaledDotProductAttention.md

```diff
@@ -0,0 +1,112 @@
+# RNNCell  {#openvino_docs_ops_sequence_ScaledDotProductAttention_13}
+
+@sphinxdirective
+
+.. meta::
+  :description: Learn about ScaledDotProductAttention-13 - a basic block for the transformer attention mechanism.
+
+**Versioned name**: *ScaledDotProductAttention-13*
+
+**Category**: *Sequence processing*
+
+**Short description**: *ScaledDotProductAttention* partially implements
+`torch.nn.functional.scaled_dot_product_attention <https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html>`__,
+omitting training-related parameter.
+
+**Detailed description**:
+
+*ScaledDotProductAttention* provides functionality according to the following pseudo-code using other operations from OpenVINO opset and ``numpy``:
+
+.. code-block:: py
+
+def ScaledDotProductAttention(query, key, value, attn_mask=None, scale=None, *, causal):
+    L, S = Gather(ShapeOf(query), -2), Gather(ShapeOf(key), -2)
+    if scale is None:
+        scale = 1.0 / Sqrt(ConvertLike(Gather(ShapeOf(query), -1), query))
+    attn_bias = Broadcast(ConvertLike(0, query), [L, S])
+    if causal:
+        attn_bias = np.triu(Broadcast(ConvertLike(-inf, query), [L, S]), k=1)
```

Comment:
in original torch pseodoce implementation used tril and logical_not, do you think it makes sense to explain that triu is equivalent replacement or align implementation?

Reply:
Just wanted to make it more compact. A careful reader can notice that this is the same algorithm, other readers don't care.